src/utils/re_frame.cljs

```diff
@@ -21,8 +21,10 @@
              (when js/goog.DEBUG
                (reset! handler-nesting-level 0))
              (when rf.interop/debug-enabled?
-               (log/debug "Handling re-frame event: "
-                          (first (interceptor/get-coeffect context :event))))
+               (do
+                 #_(log/debug "Handling re-frame event: "
+                            (first (interceptor/get-coeffect context :event)))
```

Comment:
Commented some log calls, this'll be reverted. They are being too noisy to test the back-end